### PR TITLE
Update notifications with id and timestamp

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Per [this comment](https://github.com/openshift/origin-web-console/pull/2001/files#r136993063) in [PR 2001](https://github.com/openshift/origin-web-console/pull/2001) in origin-web-console.

@spadgett @jeff-phillips-18

### CODE REVIEW

File: src/ui-services/notificationsService.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
     };
 
     var addNotification = function (notification) {
+      notification.id = notification.id || _.uniqueId('notification-');
+      // This format matches timestamp of API objects
+      notification.timestamp = moment.parseZone(new Date()).utc().format();
```
Comment (spadgett) [inline src/ui-services/notificationsService.js]: I think just

```js
new Date().toISOString();
```

does what we want, then we don't need to add moment.js to common

Comment (spadgett) [reply src/ui-services/notificationsService.js]: And it's simpler :)

Comment (benjaminapetersen) [reply src/ui-services/notificationsService.js]: True. lets do that.

**Comment by benjaminapetersen** [comment]
Just updated the filters as well to take the api object type (Normal, Warning).

File: src/ui-services/notificationsService.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
     };
 
     var addNotification = function (notification) {
+      notification.id = notification.id || _.uniqueId('notification-');
+      // This format matches timestamp of API objects
+      //notification.timestamp = moment.parseZone(new Date()).utc().format();
```
Comment (spadgett) [inline src/ui-services/notificationsService.js]: Remove commented out code

Comment (benjaminapetersen) [reply src/ui-services/notificationsService.js]: got it

**Pull request merged**